api/src/main/java/org/pmiops/workbench/compliance/ComplianceServiceImpl.java

```diff
@@ -15,14 +17,23 @@
 
   private MoodleApi api = new MoodleApi();
   private static final String RESPONSE_FORMAT = "json";
-  private static final String TOKEN = "d106acbb7eb050a10e9b200eb82b8be8";
   private static final String GET_MOODLE_ID_SEARCH_FIELD = "email";
   private static final String MOODLE_EXCEPTION = "moodle_exception";
   private static final String MOODLE_USER_NOT_ALLOWED_ERROR_CODE = "guestsarenotallowed";
+  private CloudStorageService cloudStorageService;
 
+  @Autowired
+  public ComplianceServiceImpl(
+      CloudStorageService cloudStorageService) {
```

Comment:
nit: extraneous line break here.
```suggestion
      public ComplianceServiceImpl(CloudStorageService cloudStorageService) {
```